exporters/jaeger/src/test/java/io/opentelemetry/exporters/jaeger/AdapterTest.java

```diff
@@ -91,6 +91,7 @@ public void testProtoSpan() {
     keyValue = getValue(jaegerSpan.getTagsList(), "span.status.code");
     assertNotNull(keyValue);
     assertEquals(0, keyValue.getVInt64());
```

Comment:
Why would this test case work but not be seen on the Jaeger UI?
And how is setting type tag helping it?

Reply:
The `int64` value was being set (and tested) correctly - but because the `type` was not being defined it defaulted to `string` - and therefore the Jaeger backend was reading the `string` field (which was empty) rather than the `int64` field.